operations/enterprises.md

```diff
@@ -726,6 +726,17 @@ Adds a new company to the enterprise.
 | `BillingCode` | string | optional | Billing code of the company. |
 | `AccountingCode` | string | optional | Accounting code of the company. |
 | `Address` | [Address](configuration.md#address) | optional | Address of the company. |
+| `Options` | Array of [Company option](enterprises.md#company-options)  | optional | Options of the company. |
+| `InviceDueInterval` | string | optional | Start of the interval in UTC timezone in ISO 8601 format. |
+| `Contact Person` | string | optional | Contact Person of the company. |
+| `Contact` | string | optional | Contact of the company. |
+| `Notes` | string | optional | Notes of the company. |
+| `Iata` | string | optional | Iata of the company. |
+
+#### Company options
+
+* `Invoiceable`
+* `None`
```

Comment:
That was discussed in person. There is no value `None`. The array represents all options. If there is no options the array is empty or null

Reply:
So it should be only Invoiceable?